packages/react-ui/src/widgets/OpacityControl.js

```diff
@@ -1,62 +1,61 @@
 import React from 'react';
-import { Box, Grid, InputAdornment, Slider, TextField } from '@mui/material';
-import makeStyles from '@mui/styles/makeStyles';
+import { Box, Grid, InputAdornment, Slider, TextField, styled } from '@mui/material';
 import LayerOptionWrapper from './legend/LayerOptionWrapper';
 
-const useOpacityControlStyles = makeStyles(({ spacing }) => ({
-  content: {
-    height: 'auto',
-    flex: 1
-  },
-  slider: {
-    marginTop: spacing(1)
-  },
-  input: {
-    width: spacing(8),
-    '& input': {
-      '&[type=number]': {
-        appearance: 'textfield'
-      },
-      '&::-webkit-outer-spin-button': {
-        appearance: 'none',
-        margin: 0
-      },
-      '&::-webkit-inner-spin-button': {
-        appearance: 'none',
-        margin: 0
-      }
+const Content = styled(Box)(() => ({
+  height: 'auto',
+  flex: 1
+}));
+
+const SliderWithSpace = styled(Slider)(({ theme }) => ({
+  marginTop: theme.spacing(1)
+}));
+
+const Input = styled(TextField)(({ theme }) => ({
+  width: theme.spacing(8),
+  '& input': {
+    '&[type=number]': {
+      appearance: 'textfield'
+    },
+    '&::-webkit-outer-spin-button': {
+      appearance: 'none',
+      margin: 0
+    },
+    '&::-webkit-inner-spin-button': {
+      appearance: 'none',
+      margin: 0
     }
-  },
-  noMargin: {
+  }
+}));
+
+const InputUnit = styled(InputAdornment)(({ theme }) => ({
+  '& 	.MuiInputAdornment-positionEnd': {
     margin: 0
   }
 }));
 
 export default function OpacityControl({ opacity, onChangeOpacity }) {
-  const classes = useOpacityControlStyles();
   const handleTextFieldChange = (e) => {
     const newOpacity = parseInt(e.target.value || '0');
     onChangeOpacity(Math.max(0, Math.min(100, newOpacity)) / 100);
   };
 
   return (
     <LayerOptionWrapper label='Opacity'>
-      <Box className={classes.content}>
+      <Content>
         <Grid container spacing={2} direction='row' alignItems='center'>
           <Grid item xs>
```

Comment:
If you do this: `<Grid item xs mt={1}>`


Probably you don't need the `SliderWithSpace` custom component... which seems overkill just for adding spacing.